### PR TITLE
[ADD] multi-company record rule for MIS Budgets

### LINKED ISSUE #27: Basic multi-company support in mis_builder_budget
This is for basic multi company support (a simple record rule). #34 is for budget consolidation.

### PR DESCRIPTION
**Pull request by sbidoul**
Fixes #27

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

File: mis_builder_budget/security/mis_budget.xml [hunk 1]
```diff
@@ -24,4 +24,11 @@
         <field name="perm_unlink" eval="1"/>
     </record>
 
+    <record id="mis_budget_multi_company_rule" model="ir.rule">
+        <field name="name">mis.budget multi company</field>
+        <field name="model_id" ref="model_mis_budget"/>
+        <field name="global" eval="True"/>
```
Comment (adrienpeiffer) [inline mis_builder_budget/security/mis_budget.xml]: This one is computed

File: mis_builder_budget/security/mis_budget_item.xml [hunk 2]
```diff
@@ -24,4 +24,11 @@
         <field name="perm_unlink" eval="1"/>
     </record>
 
+    <record id="mis_budget_item_multi_company_rule" model="ir.rule">
+        <field name="name">mis.budget.item multi company</field>
+        <field name="model_id" ref="model_mis_budget_item"/>
+        <field name="global" eval="True"/>
```
Comment (adrienpeiffer) [inline mis_builder_budget/security/mis_budget_item.xml]: This one is computed

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

File: mis_builder_budget/security/mis_budget.xml (hunk 1 above)
Comment (sbidoul) [reply mis_builder_budget/security/mis_budget.xml]: @adrienpeiffer what do you mean?

Comment (adrienpeiffer) [reply mis_builder_budget/security/mis_budget.xml]: The ```global``` field of ```ir.rule``` is computed. So, you shouldn't set this one.

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Pull request merged**